features

```diff
@@ -1,2 +1,2 @@
-use-npm-ci=0
+use-npm-ci=50
```

Comment:
note: this value doesn't match the title

Reply:
thanks @jmorrell - updated to reflect title